src/components/Metrics/Metrics-i18n.json

```diff
@@ -11,9 +12,10 @@
     "connection_attempt_rate_yaxis": "Connections/second",
     "log_size_per_partition": "Log size per partition",
     "log_size_per_partition_help_text": "Topic partition size is the log size for each partition in a selected topic. This metric enables you to assess the amount of data used in one or more topic partitions. To reduce the topic partition size, you can decrease the retention time or the retention size, or modify the cleanup policy as needed. You can also increase the number of partitions for the topic.",
-    "incoming_bytes_per_topic": "Incoming bytes per topic",
-    "outgoing_bytes_per_topic": "Outgoing bytes per topic",
-    "total_bytes": "Total bytes",
+    "all_topics": "all topics",
+    "incoming_bytes": "Incoming bytes ({{topic}})",
+    "outgoing_bytes": "Outgoing bytes ({{topic}})",
+    "total_bytes": "Bytes incoming and outgoing",
```

Comment:
I see that you have the right text here, but it isn't showing up in the storybook example: 
![image](https://user-images.githubusercontent.com/21063328/144087209-79b7df91-3738-4d93-bac3-6f674caf5b7f.png)

Reply:
Oh, nevermind. I just thought to check if I was looking at the latest storybook. I wasn't. 🙃